kolibri/core/auth/api.py

```diff
@@ -326,7 +332,12 @@ def filter_exclude_member_of(self, queryset, name, value):
 
     def filter_exclude_coach_for(self, queryset, name, value):
         return queryset.exclude(
-            Q(roles__in=Role.objects.filter(kind=role_kinds.COACH, collection=value))
+            Q(
+                roles__in=Role.objects.filter(
+                    Q(kind=role_kinds.COACH) | Q(kind=role_kinds.ASSIGNABLE_COACH),
+                    collection=value,
+                )
+            )
```

Comment:
We can always revert this change if not necessary, but thought It would be great to make at this point, lest we forget!